logstash-core/src/test/java/org/logstash/ackedqueue/TestSettings.java

```diff
@@ -9,7 +9,7 @@
 
 public class TestSettings {
 
-    public static Settings getSettings(int capacity) {
+    public static Settings volatileQueueSettings(int capacity) {
```

Comment:
> getSettingsCheckpointFilePageMemory

I kind of want to name all the methods this, now. haha.

Thanks for the clarification <3